.github/workflows/markdown-linting/.markdownlint.json

```diff
@@ -0,0 +1,11 @@
+{
+  "no-inline-html": false,
+  "no-bare-urls": false,
```

Comment:
I'm curious on this one - what are the failures? Is it inside code or something?

Reply:
By default, `markdownlint` wants us to replace `hello https://example.com world` with `hello <https://example.com> world`, assuming that URL parsing may be unstable (e.g. when there is `?` at the very end or some UTF8 chars in the middle). Rule details can be found here:
https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md

I tried living both with and without this rule, but TBH did not find any practical value in it. When the rule is enabled, contributors get confused because wrapping URLs with `<>` is not that common or intuitive.

IMHO we can start with the rule being disabled and reconsider the decision later on if raw links bring issues.

Reply:
Using `<>` also prevents the links from being rendered as links, if I'm not wrong.